custom_reports/http_api/definitions/query_request.yaml

```diff
@@ -28,14 +28,22 @@ properties:
     example: [{"column_id":"device_type"},{"column_id":"revenue","transformation_id":"sum"}]
   date_from:
     type: string
-    format: date
-    description: Start date for the query (inclusive). Cannot be used with relative_date field at
-      the same time. Mandatory if ``relative_date`` is not used.
+    oneOf:
+      - format: date
+        description: Start date for the query (inclusive).
+      - format: datetime
+        description: Start datetime for the query (inclusive, in website's timezone).
```

Comment:
if we use "datetime" here, it's worth unifying it in "Start date (with time) -> Start datetime" also. alternatively we could consult the copy with Joanna

Reply:
and also i would use ``` `relative_date` ``` where possible, in one place there is ``` `relative_date` ``` in both places, in some cases only in one place.

Reply:
Cannot be used with ``` ``relative_date`` ``` field at the same time.
      Mandatory if ``` ``relative_date`` ``` is not used.

vs

Cannot be used with relative_date field at the same time.
      Mandatory if ``` ``relative_date`` ``` is not used.